src/Motor.Extensions.Diagnostics.Telemetry/TelemetryHostBuilderExtensions.cs

```diff
@@ -33,78 +29,61 @@ public static IMotorHostBuilder ConfigureOpenTelemetry(this IMotorHostBuilder ho
     {
         Activity.DefaultIdFormat = ActivityIdFormat.W3C;
         Activity.ForceDefaultIdFormat = true;
-        hostBuilder
-            .ConfigureServices((hostContext, services) =>
+        hostBuilder.ConfigureServices((hostContext, services) =>
+        {
+            UsedExporter? usedExport;
+            if (hostContext.Configuration.GetSection(OtlpExporter).Exists())
             {
-                var usedExport = UsedExporter.Jaeger;
-                if (hostContext.Configuration.GetSection(OtlpExporter).Exists())
-                {
-                    services.Configure<OtlpExporterOptions>(hostContext.Configuration.GetSection(OtlpExporter));
-                    usedExport = UsedExporter.Oltp;
-                }
-                else
-                {
-                    services.Configure<JaegerExporterOptions>(hostContext.Configuration.GetSection(JaegerExporter));
-                }
+                services.Configure<OtlpExporterOptions>(hostContext.Configuration.GetSection(OtlpExporter));
+                usedExport = UsedExporter.Oltp;
+            }
+            else
+            {
+                services.Configure<JaegerExporterOptions>(hostContext.Configuration.GetSection(JaegerExporter));
+                usedExport = UsedExporter.Jaeger;
```

Comment:
We could get rid of the Jeager stuff or?

Reply:
Can we? Is there a reason we should stop supporting Jaeger?

Reply:
I think otlp is the more common protocol. And you can convert traces in both directions.

Reply:
In that case, I would still prefer to remove Jaeger support in another PR.

Reply:
Sounds great.